src/jobservice/period/basic_scheduler.go

```diff
@@ -155,7 +155,7 @@ func (bs *basicScheduler) UnSchedule(policyID string) error {
 			// Mark job status to stopped to block execution.
 			// The executions here should not be in the final states,
 			// double confirmation: only stop the can-stop ones.
-			if job.RunningStatus.Compare(job.Status(e.Info.Status)) >= 0 {
+			if !job.RunningStatus.Before(job.Status(e.Info.Status)) {
```

Comment:
>=0 means Before() or Equal()?

Reply:
![image](https://user-images.githubusercontent.com/37969177/100037479-285d0680-2e3d-11eb-849d-cb5a4a5449eb.png)
the old logic is above， so if a.Before(b) is true， means a is earlier than b ( in old logic a.Compare(b) < 0)

Reply:
you mean I should write like this?
```
job.RunningStatus.After(job.Status(e.Info.Status)) || job.RunningStatus.Equal(job.Status(e.Info.Status))
```

Reply:
Yup! >=0 means your `Before` and `Equal `

<=0 means your `After` and `Equal`.

@shaobo322